lib/addons/package.json

```diff
@@ -33,6 +33,7 @@
     "@storybook/client-logger": "6.0.0-alpha.20",
     "@storybook/core-events": "6.0.0-alpha.20",
     "@storybook/router": "6.0.0-alpha.20",
+    "@storybook/theming": "6.0.0-alpha.20",
```

Comment:
Ohh, I don't think we can have this dependency can we? `@storybook/theming` depends on `emotion`, no? 

Or is it OK because we don't require the files that do that in addons?

Reply:
It's only used for types, it should not appear in the dist files.